src/actions/userActions.ts

```diff
@@ -42,6 +46,13 @@ export namespace User {
       type = Logout.type;
     }
 
+    export class ChangeOrg {
+      static type = 'CHANGE_ORG';
```

Comment:
It's best to prefix redux actions to avoid naming clashes and just to make debugging a little easier.

```suggestion
      static type = 'USER_CHANGE_ORG';
```